examples/simple_dict.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# coding=utf-8
+"""
+Simple demonstration of TableFormatter with a list of dicts for the table entries.
```

Comment:
Adding unit tests related to the two things that got fixed and the new feature would be highly desirable.